lightly_studio/.env.example

```
@@ -33,6 +33,7 @@ EXAMPLES_PASCALVOC_CATEGORIES_JSON_PATH="datasets/voc2012_10_images/class_id_to_
 EXAMPLES_MIDV_PATH="datasets/midv_2020_10_samples"
 EXAMPLES_ACTIVITYNET_VIDEOS_PATH="datasets/activitynet_10_videos/videos"
 EXAMPLES_ACTIVITYNET_JSON_PATH="datasets/activitynet_10_videos/activity_net.v1-3.min.json"
+EXAMPLES_TIMED_CAPTIONS_VIDEO_PATH="datasets/activitynet_10_videos/videos/F2x2fynkbAQ.mp4"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)